js/index.js

```diff
@@ -16,4 +16,44 @@ for (i=0; i < skills.length; i++) {
     skill.innerText = skills[i];
     skillsList.appendChild(skill)
 
-}
\ No newline at end of file
+}
+
+// messages event listener
+const messageForm = document.querySelector('form');
+
+messageForm.addEventListener('submit', (e) => {
+    e.preventDefault();
+
+    const name = e.target.name.value;
+    const email = e.target.email.value;
+    const message = e.target.message.value;
+
+    // console.log(name, email, message);
+
+    const messageSection = document.getElementById('messages'); //display message in list 
+
+  
+
+    const messageList = messageSection.querySelector('ul');
+    const newMessage = document.createElement('li');
+    newMessage.innerHTML = `<div> 
+                           <span class="strong">${message}</span> 
+                           <p><a class="link" href="mailto:${ email}">${name}</a> &nbsp;</p> 
+                           </div>`
+
+    const removeButton = document.createElement('button');
+    removeButton.innerText = 'remove';
+    removeButton.type = 'button';
+    
+    removeButton.addEventListener('click', (e) => {
+        const entry = e.target.parentNode;
+        messageList.removeChild(entry);
```

Comment:
This works well :+1:  Another way to do it is: `entry.remove();`

https://developer.mozilla.org/en-US/docs/Web/API/Element/remove